lib/api/programsLogic.ts

```diff
@@ -9,7 +9,7 @@ export const getProgramsHandler = async (req: NextApiRequest, res: NextApiRespon
 
 export const postProgramsHandler = async (req: NextApiRequest, res: NextApiResponse) => {
   const validatedProgram = validateCreateProgramsRequest(req.body);
-  await prisma.program.create({ data: validatedProgram });
+  await prisma.program.create({ data: req.body });
```

Comment:
Looks like we aren't using the validatedProgram const that was created? Was this a typo or are do we want to remove the validateCreateProgramsRequest as well?